xbd/inference.py

```diff
@@ -154,17 +304,36 @@ def prediction(input_img_path: str) -> List[Instance]:
     instance = Instance(
       id=unique_id,
       mask_url=f'/static/{unique_id}.png',
+      label='building',
       box_coordinates=[
-        (matrix_components[i][2], matrix_components[i][3]),
-        (matrix_components[i][4], matrix_components[i][3]),
-        (matrix_components[i][4], matrix_components[i][5]),
-        (matrix_components[i][2], matrix_components[i][5])
+          (matrix_components[i][2], matrix_components[i][3]),
+          (matrix_components[i][4], matrix_components[i][3]),
+          (matrix_components[i][4], matrix_components[i][5]),
+          (matrix_components[i][2], matrix_components[i][5])
```

Comment:
아래의 제안 코드처럼 형식을 유지해야합니다.

```suggestion
         (matrix_components[i][2], matrix_components[i][3]),
         (matrix_components[i][4], matrix_components[i][3]),
         (matrix_components[i][4], matrix_components[i][5]),
         (matrix_components[i][2], matrix_components[i][5])
```